packages/datetime/test/dateInputTests.tsx

```diff
@@ -14,6 +14,14 @@ import { padWithZeroes } from "../src/common/utils";
 import { Classes, DateInput } from "../src/index";
 
 describe("<DateInput>", () => {
+    it("handles null inputs without crashing", () => {
+        mount(<DateInput value={null} />);
+    });
+
+    it("handles string inputs without crashing", () => {
+        mount(<DateInput value="1988-08-07 11:01:12" />);
```

Comment:
@adidahiya this is not valid usage of the component and throws a compiler error in tests. should we add `string` to `value` types or cast this `as any`?